MetaMask/metamask-extension
# Updating all Box background and border color instances

Fixes: #13774 

Explanation: Updating all Box background and border color instances with new design token colors. I'm also updating any components that I have touched with the Box updates so no components are only partially updated. This is a large PR but once this is merged I create more digestible PRs for one component at a time. 

Manual testing steps:  
  - Go to [latest build of storybook](https://502375-42009758-gh.circle-artifacts.com/0/storybook/index.html) or run `yarn storybook`
  - Search `Callout`  check in light mode and dark mode using 🌙 icon in navbar
  - Search `Chip` check in light mode and dark mode using 🌙 icon in navbar
  - Search `SnapsAuthorshipPill` check in light mode and dark mode using 🌙 icon in navbar
  - Search `SnapSettingsCard` check in light mode and dark mode using 🌙 icon in navbar
  - Search `TextArea`  check in light mode and dark mode using 🌙 icon in navbar
  
## Screenshots
Call out

![Callout](https://user-images.githubusercontent.com/8112138/156618520-adfadb19-d09d-42b7-8f43-a646e1e9a87e.png)

Chip

![Chip](https://user-images.githubusercontent.com/8112138/156618546-1480d9ea-9c7c-48f4-8da9-39a8d4e12924.png)

SnapsAuthorshipPill

![SnapsAuthorshipPill](https://user-images.githubusercontent.com/8112138/156618591-09eb5b0a-f882-4f1d-9448-a69c35c095ff.png)

SnapSettingsCard

![SnapSettingsCard](https://user-images.githubusercontent.com/8112138/156618594-e1e4636f-1ae9-4be6-a939-4b1a49d63c24.png)

TextArea

![Textarea](https://user-images.githubusercontent.com/8112138/156618596-a4c7bb0b-6d39-4c86-9281-2c11cb6d904d.png)
